cli/Valet/CommandLine.php

```diff
@@ -74,7 +74,11 @@ function runCommand($command, callable $onError = null)
     {
         $onError = $onError ?: function () {};
 
-        $process = new Process($command);
+        if (method_exists(Process::class, 'fromShellCommandline')) {
```

Comment:
Much as I try to avoid comments by letting the code speak for itself, this is probably one of those cases where we should add a code comment saying why we're doing this, particularly since at some point we'll want to drop the deprecated code altogether, and having the right Symfony version number documented will be helpful.

Reply:
Agreed!